docs/tutorial/node/src/from_one_to_multiple_tiles.md

```diff
@@ -1,34 +1,40 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
 # From One To Multiple Tiles
 
-After modeling a single tile, let's create a grid of them. For the grid to be our game board, we need two features:
+After modeling a single tile, this step creates a grid of them. For the grid to be a game board, you need two features:
+
+1. **A data model**: An array created as a Rust model, where each element describes the tile data structure, such as:
+
+    - URL of the image
+    - Whether the image is visible
+    - If the player has solved this tile.
+
+2. A way of creating multiple instances of the tiles.
 
-1. A data model: This shall be an array where each element describes the tile data structure, such as the
-   url of the image, whether the image shall be visible and if this tile has been solved. We modify the model
-   from JS code.
-2. A way of creating many instances of the tiles, with the above `.slint` markup code.
+With Slint you declare an array of structures based on a model using square brackets. Use a <span class="hljs-keyword">for</span> loop
+to create multiple instances of the same element.
 
-In Slint we can declare an array of structures using brackets, to create a model. We can use the <span class="hljs-keyword">for</span> loop
-to create many instances of the same element. In `.slint` the for loop is declarative and automatically updates when
-the model changes. We instantiate all the different <span class="hljs-title">MemoryTile</span> elements and place them on a grid based on their
-index with a little bit of spacing between the tiles.
+With Slint the for loop is declarative and automatically updates when
+the model changes. The loop instantiates all the <span class="hljs-title">MemoryTile</span> elements and places them on a grid based on their
+index with spacing between the tiles.
 
-First, we copy the tile data structure definition and paste it at top inside the `memory.slint` file:
+First, add the tile data structure definition at the top of the `memory.slint` file:
 
 ```slint
 {{#include ../../rust/src/main_multiple_tiles.rs:tile_data}}
 ```
 
-Next, we replace the _export component <span class="hljs-title">MainWindow</span> inherits Window { ... }_ section at the bottom of the `memory.slint` file with the following snippet:
+Next, replace the _export component <span class="hljs-title">MainWindow</span> inherits Window { ... }_ section at the bottom of the `slint!` macro with the following:
```

Comment:
This seems wrong, referring to the slint macro from Rust when this is the Node.js quickstart.